201-blob-storage-private-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,915 @@
+      "metadata": {
+        "description": "Specifies the name of the private link to Azure Database for PostgreSQL."
+      }
+    }
+  },
+  "variables": {
+    "nicName": "[concat(parameters('vmName'), 'Nic')]",
+    "nsgName": "[concat(parameters('subnetName'), 'SubnetNsg')]",
+    "publicIPAddressName": "[concat(parameters('vmName'), 'PublicIp')]",
+    "publicIPAddressType": "Dynamic",
+    "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]",
+    "workspaceId": "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspaceName'))]",
+    "subnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworkName'), parameters('subnetName'))]",
+    "publicIpId": "[resourceId('Microsoft.Network/publicIPAddresses', variables('publicIPAddressName'))]",
+    "nicId": "[resourceId('Microsoft.Network/networkInterfaces', variables('nicName'))]",
+    "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+    "nsgId": "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
+    "vmId": "[resourceId('Microsoft.Compute/virtualMachines', parameters('vmName'))]",
+    "linuxConfiguration": {
+      "disablePasswordAuthentication": true,
+      "ssh": {
+        "publicKeys": [
+          {
+            "path": "[concat('/home/', parameters('adminUsername'), '/.ssh/authorized_keys')]",
+            "keyData": "[parameters('adminPasswordOrKey')]"
+          }
+        ]
+      },
+      "provisionVMAgent": true
+    },
+    "publicDNSZoneForwarder": "[concat('.blob.', environment().suffixes.storage)]",
+    "blobServicePrimaryEndpoint": "[concat(parameters('storageAccountName'), variables('publicDNSZoneForwarder'))]",
+    "privateDnsZoneName": "[concat('privatelink', variables('publicDNSZoneForwarder'))]",
+    "privateDnsZoneId": "[resourceId('Microsoft.Network/privateDnsZones', variables('privateDnsZoneName'))]",
+    "privateEndpointId": "[resourceId('Microsoft.Network/privateEndpoints', parameters('privateEndpointName'))]",
+    "privateEndpointGroupId": "blob",
+    "customScriptId": "[resourceId('Microsoft.Compute/virtualMachines/extensions', parameters('vmName'), 'CustomScript')]",
+    "omsAgentForLinuxId": "[resourceId('Microsoft.Compute/virtualMachines/extensions', parameters('vmName'), 'LogAnalytics')]",
+    "scriptFileUri": "[uri(parameters('_artifactsLocation'), concat('scripts/', parameters('scriptFileName'), parameters('_artifactsLocationSasToken')))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2019-06-01",
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[parameters('storageAccountName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "Standard_LRS"
+      },
+      "kind": "StorageV2",
+      "properties": {
+        "accountType": "[parameters('storageAccountType')]"
```

Comment:
Or another option would be to force the storage account to Standard_LRS (like it is currently inadvertently) and use the one account for everything - sample works in that mode, just not a flexible as it would be otherwise.